public/docs/ts/latest/guide/architecture.jade

```diff
@@ -267,29 +255,27 @@ block ts-decorator
   For example, if an app's  HTML contains `<hero-list></hero-list>`, then
   Angular inserts an instance of the `HeroListComponent` view between those tags.
 
-  - `templateUrl`: address of this component's template, which we showed [above](#templates).
+  - `templateUrl`: address of this component's HTML template, shown [above](#templates).
 
   - `directives`: !{_array} of the components or directives that *this* template requires.
-  We saw in the last line of our template that we expect Angular to insert a `HeroDetailComponent`
+  In the last line of `hero-list.component.html`, Angular inserts a `HeroDetailComponent`
   in the space indicated by `<hero-detail>` tags.
-  Angular will do so only if we mention the `HeroDetailComponent` in this `directives` !{_array}.
+  Angular will do so only if the `HeroDetailComponent` in this `directives` !{_array}.
```

Comment:
Drop "the" before the classname. ("the _classname_" is OK if you're talking about an object, but here you're talking about not an instance of the class but the actual class, which is in the array.)